qemu/tests/cfg/cpu_info_check.cfg

```diff
@@ -0,0 +1,8 @@
+- cpu_info_check:
+    type = cpu_info_check
+    start_vm = no
+    cpu_model_3_1_0 = 'Cascadelake-Server Icelake-Client Icelake-Server KnightsMill'
+    cpu_model_8_2_0 = 'Cascadelake-Server-noTSX Icelake-Client-noTSX Icelake-Server-noTSX'
+    cpu_model_8_2_0 += ' Skylake-Client-noTSX-IBRS Skylake-Server-noTSX-IBRS'
+    cpu_model_8_2_1 = 'Cooperlake'
+    cpu_model_8_3_0 = 'EPYC-Rome'
```

Comment:
@vivianQizhu Hi, sorry for bothering you again in cfg. I think we can set qemu2.12 as one base. and since we also need to the old CPU models we can add the old CPU models when we check newer qemu versions like the following.
you can choose your own way to do this if you have one better way. Thanks.

```
cpu_model_2_12_0 = 'Skylake-Client Skylake-Server'
cpu_model_3_1_0 = cpu_model_2_12_0 + 'Cascadelake-Server Icelake-Client Icelake-Server KnightsMill'
cpu_model_8_2_0 = cpu_model_3_1_0 + ...
....
```

Reply:
@nanliu-r `cpu_model_3_1_0 = cpu_model_2_12_0 + '...'` will make code logic more complicated, the version between 2.12.0 - 3.1.0 will need to remove models from scope (cpu_model_3_1_0 - cpu_model_2_12_0), `+` in cfg them `-` in code is not necessary I think.

Reply:
@nanliu-r And please check the code carefully, we do check old CPU models on both old and new host, we only remove these new models when the qemu version is lower, for those new host we check all of them. https://github.com/autotest/tp-qemu/pull/2321/files#diff-6b54a2c5438a5b74a02879a8bf3611fdR49